src/firmware/app/control/trajectory_planner.h

```diff
@@ -99,9 +121,9 @@ typedef struct FirmwareRobotPathParameters
  *
  * @return The outcome of the trajectory generation. Returns OK if trajectory is valid.
  */
-TrajectoryPlannerGenerationStatus
+TrajectoryPlannerGenerationStatus_t
 app_trajectory_planner_generateConstantArcLengthTrajectory(
```

Comment:
```suggestion
app_trajectory_planner_generateConstantArcLengthPositionTrajectory(
```
Would this be more accurate?